lib/features/give/pages/home_page.dart

```diff
@@ -153,7 +153,8 @@ class _HomePageState extends State<HomePage> {
                 if (!auth.user.needRegistration || auth.user.mandateSigned) {
                   return;
                 }
-                _buildNeedsRegistrationDialog(context);
+                final isUS = auth.user.country == Country.us.countryCode;
```

Comment:
its not needed to pass this information. User object is also available in the _build... method

Reply:
Improved